src/fields.jl

```diff
@@ -189,6 +210,8 @@ function 𝐼y!(output::Field2D{Centre, Centre}, input::Field2D{Centre, Face, Gr
     for j in 1:ny-1, i in 1:nx
```

Comment:
???

Reply:
Yes, my bad. It needed more work. Have a look at my commit now. I think I've sorted everything. There are some tests which I need to add, but apart from that, everything should be okay.